### PR TITLE
boards/nucleo-l4r5zi: doc improvements

### PR DESCRIPTION
**Pull request by krzysztof-cabaj**
### Contribution description

Improvements to STM Nucleo l4r5zi documentation:

- Add board image,
- Add MCU table - similar to those, for example for, Nucleo F103RB, F302R8 or F446RE,
- Add information concerning accessing RIOT shell (see Issue #17453) 

### Testing procedure

```
make doc
xdg-open doc/doxygen/html/group__boards__nucleo144-l4r5.html
```

### Issues/PRs references

Issue #17453

### CODE REVIEW

**Review by benpicco — APPROVED** [review]

File: boards/nucleo-l4r5zi/doc.txt [hunk 1]
```diff
@@ -8,6 +8,31 @@
...
 
+## Hardware
+
+![Nucleo144 L4R5ZI](https://www.st.com/bin/ecommerce/api/image.PF264781.en.feature-description-include-personalized-no-cpn-large.jpg)
+
+### MCU
+
+| MCU          | STM32L4R5ZI                                        |
+|:-------------|:---------------------------------------------------|
+| Family       | ARM Cortex-M4                                      |
```
Comment (benpicco) [inline boards/nucleo-l4r5zi/doc.txt:19]: Small nit: the core comes with a FPU
```suggestion
| Family       | ARM Cortex-M4F                                     |
```

Comment (krzysztof-cabaj) [reply boards/nucleo-l4r5zi/doc.txt]: Thank. I add F to the MCU family and additional row in the table concerning FPU - like in [nucleo-l552ze-q](https://doc.riot-os.org/group__boards__nucleo-l552ze-q.html).

**Comment by benpicco** [comment]
Please squash

**Comment by krzysztof-cabaj** [comment]
Squashed. Thanks!

**Pull request merged**